libdnf/module/ModulePackage.cpp

```diff
@@ -154,16 +162,27 @@ std::string ModulePackage::getNameStreamVersion() const
     return ss.str();
 }
 
+std::string ModulePackage::getRepoID() const
```

Comment:
Why are you returning a copy of string instead of reference `const std::string &`?